app/views/reorder_timeline_entries/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<%
+  links = [
+    {
+      text: 'Coronavirus pages',
+      href: coronavirus_pages_path
+    },
+    {
+      text: "#{@coronavirus_page.name}",
+      href: coronavirus_page_path(slug: @coronavirus_page.slug)
+    },
+    {
+      text: "Reorder timeline entries"
+    },
+  ]
+%>
+<% content_for :breadcrumbs, render('shared/steps/step_breadcrumb', links: links) %>
+<% content_for :title, formatted_title(@coronavirus_page)%>
+<% content_for :context, "Reorder timeline entries" %>
+
+<div class="covid-manage-page govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render_markdown "Use the up/down buttons on the right to reorder the timeline entries, or click and hold on a section to reorder using drag and drop." %>
+
+    <%= form_with method: :put, url: reorder_coronavirus_page_timeline_entries_path do |form| %>
+      <input type="hidden" name="timeline_entry_order_save" id="js_order_save" value="" />
```

Comment:
This reordering pattern was copied wholesale from[ Step by Steps](https://github.com/alphagov/collections-publisher/blob/master/app/views/step_by_step_pages/reorder.html.erb#L32) so I think it might have pre-dated Content Publisher.

I think you're right though that we should consider adding a reorder component to govuk_publishing_components in a follow-up story.